core/templates/dev/head/pages/exploration-player-page/learner-experience/learner-answer-info-card.directive.html

```diff
@@ -0,0 +1,53 @@
+<div class="learner-answer-info-card">
+  <!-- <div class="conversation-skin-learner-answer-container">
+    <angular-html-bind class="oppia-rte-viewer conversation-skin-learner-answer-content current-answer" html-data="$ctrl.displayCurrentAnswer()">
+    </angular-html-bind>
+  </div>
+  <p class="question-content" translate="I18N_SOLICIT_ANSWER_DETAILS_QUESTION"></p> -->
+  <div class="enter-answer-details">
+    <textarea class="form-control" ng-model="$ctrl.answerDetails" rows="4" placeholder="Enter here"></textarea>
+  </div>
+  <div class="submit-button">
+    <md-button class="oppia-learner-confirm-button" ng-disabled="!($ctrl.answerDetails)" ng-click="$ctrl.submitLearnerAnswerInfo()">SUBMIT</md-button>
+  </div>
+</div>
+<style>
+  .learner-answer-info-card {
+    background: #fff;
+    border-radius: 2px;
+    padding-top: 20px;
+    text-align: left;
+    margin-left: 87px;
+  }
+
+  .learner-answer-info-card textarea {
+    resize: vertical;
+  }
+
+  .learner-answer-info-card .enter-answer-details {
+    padding: 20px;
+  }
+
+  .learner-answer-info-card .submit-button {
+    position: relative;
+    left: 80%;
+  }
+
+  .learner-answer-info-card .question-content {
+    position: relative;
+    left: 5%;
+  }
+
+  .conversation-skin-learner-answer-container {
```

Comment:
Is this still needed, given that the directive is commented out above?

Reply:
Done